library/src/main/java/de/westnordost/osmfeatures/FeatureDictionary.java

```diff
@@ -319,7 +319,7 @@ private FeatureTermIndex createNameIndex(List<Locale> locales)
 	{
 		return new FeatureTermIndex(featureCollection.getAll(locales), feature -> {
 			if (!feature.isSearchable()) return Collections.emptyList();
-			String name = feature.getCanonicalName();
+			String name = feature.getCanonicalName().replaceAll("\\(|\\)", "");

```

Comment:
```suggestion
			String name = feature.getCanonicalName().replaceAll("[()]", "");
```
a little shorter and easier to read

Reply:
Thank you, I'm no regex wizard.

I was also expecting this to get turned into a String.Join with an array of removable strings, but after looking through the English files, open and close brackets were the only ones. I expect this will grow with other languages though.